CHANGELOG.md

```diff
@@ -3,6 +3,19 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [4.0.0] - 2024-03-04
+
+## Changed
+
+- Update govuk-frontend to 5.0 (https://github.com/alphagov/govuk-frontend/blob/main/CHANGELOG.md#500-breaking-release)
+- Remove IE polyfills
+- Update favicons
+- Update site logo
+- Update markup in line with 5.0
+- Update build file paths
+- Set jquery as dependency for main.js
+- Update govuk-frontend to 4.0
```

Comment:
@sbrody I'd probably be tempted to remove this line - it's true that we did upgrade to 4.0 as part of the work to get to 5.0, but if I think explicitly mentioning it here muddies the water in terms of what version of govuk-frontend this release of the theme actually uses.

Reply:
Ok - makes sense - will remove